orchard-ws/conf/routes

```diff
@@ -24,6 +24,7 @@ PUT     /v1/workflow/:id            controllers.WorkflowController.activate(id:
 DELETE  /v1/workflow/:id            controllers.WorkflowController.delete(id: String)
 
 ->      /v1/workflows               routers.WorkflowsRouter
+->      /v1/stats                   routers.StatsRouter
```

Comment:
@realstraw is the intention for this route to be used in adhoc fashion?

Reply:
adhoc or if people want to use it to build some kind of admin panel or dashboard

Reply:
@realstraw good point, this would be very useful for admin/mission control panel, and a pattern to follow to expand in future if needed